src/parsers/pluscal.ts

```diff
@@ -124,21 +125,25 @@ export class TranspilerStdoutParser extends ProcessOutputHandler<DCollection> {
             return false;
         }
 
-        const matcher = /^\s\s([A-Za-z0-9_]+) at line \d+, column \d+$/g.exec(line);
+        const matcher = /^\s\s([A-Za-z0-9_]+) at line \d+, column \d+/g.exec(line);
```

Comment:
Note the removal of the `$` at the end, so it will match as long as the string has `matcher` as a prefix. This is only called if we're in `label parsing mode` so won't affect parsing any other output.